addons/purchase_stock/models/purchase_order_line.py

```diff
@@ -43,7 +43,7 @@ def _get_po_line_moves(self):
             moves = moves.filtered(lambda r: fields.Date.context_today(r, r.date) <= self._context['accrual_entry_date'])
         return moves
 
-    @api.depends('move_ids.state', 'move_ids.product_uom_qty', 'move_ids.product_uom')
+    @api.depends('move_ids.state', 'move_ids.product_uom_qty', 'move_ids.product_uom', 'move_ids.quantity')
```

Comment:
It's in fact due to this PR https://github.com/odoo/odoo/pull/130342
We should have done the same change https://github.com/odoo/odoo/pull/130342/files#diff-c728b1f95e4d88de6354fa24c888c5a70e7c435f43002b7ee1c87238e6cab80eL150-L151

Reply:
@ethanrobv So you can remove `'move_ids.product_uom_qty` :P